docs/theming-flatpak.rst

```diff
@@ -0,0 +1,46 @@
+Theming Flatpak
+===============
+
+Flatpak applications cannot directly use the system theme. This happens because all flatpaks are sandboxed and do not have the ability to use data files or libraries in ``/usr`` (where system themes are typically located). This is done so that the format is portable across a wide range of distributions in a reproducible environment, but has the downside of not being able to pick up system themes. The solution to this was to package GTK themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as `extensions <https://github.com/flatpak/flatpak/wiki/Extensions>`_ to the Gnome runtime.
```

Comment:
This can probably be simplified a bit, since this is no longer a blog post and is within the flatpak docs I don't think we need to re-describe what flatpak is.

Also at the end the Gtk extension point is in the Freedesktop runtime and Qt one in the KDE one.